### PR TITLE
🔍 Read Operations for a Person

### PR DESCRIPTION
**Pull request by usamazansari**
Implemented the following queries:

1. `getPeople()` - Returns all the nodes tagged with the label `:PERSON`
2. `getPerson(person: Partial<Person>)` - Returns the person who is matched

### CODE REVIEW

File: libs/api/features/trip/src/lib/imports/mock/index.ts [hunk 1]
```diff
@@ -1 +1,4 @@
-export { GoaUsers } from '@itinerary-assistant/api/features/user';
+export {
```
Comment (usama251993) [inline libs/api/features/trip/src/lib/imports/mock/index.ts:1]: No longer needed, it is available in the database

**Review by usama251993 — APPROVED** [review]
Resolve this comments in `neo4j/base`

File: libs/api/database/entities/src/lib/entities/user/person.entity.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  SocialConnection as SocialConnectionModel
+} from '../imports/models';
+import {
+  Address as AddressEntity,
+  Identification as IdentificationEntity,
+  Demographics as DemographicsEntity,
+  SocialConnection as SocialConnectionEntity
+} from '.';
+
+@ObjectType()
```
[the remark this answers is not in the document]
Comment (usamazansari) [reply libs/api/database/entities/src/lib/entities/user/person.entity.ts:18]: Resolved in https://github.com/usamazansari/itinerary-assistant/commit/0420b8287be1d228b70140a4373bd19574a11048

File: libs/api/database/graphql/src/lib/graphql.module.ts [hunk 3]
```diff
@@ -4,11 +4,7 @@ import { ApolloDriver, ApolloDriverConfig } from '@nestjs/apollo';
 
 import { join } from 'path';
 
-import {
-  SanityRepository,
-  SanityResolver,
-  SanityService
-} from './sanity';
+import { SanityRepository, SanityResolver, SanityService } from './sanity';
```
[the remark this answers is not in the document]
Comment (usamazansari) [reply libs/api/database/graphql/src/lib/graphql.module.ts:7]: Resolved in https://github.com/usamazansari/itinerary-assistant/commit/40ae90b987b137782d945aa05998412a64ac7ea2

**Pull request merged**